packages/ecc-client-lit-ga4gh-tes/src/components/create-run/create-run.ts

```diff
@@ -0,0 +1,79 @@
+import { html, LitElement } from "lit";
+import "@elixir-cloud/design";
+
+export class CreateRun extends LitElement {
+  fields = [
```

Comment:
How are these fields going to be used, will this be the form schema and dev will iterate through it to create form?

Reply:
Yes!